edb/pgsql/common.py

```diff
@@ -202,9 +202,22 @@ def get_objtype_backend_name(id, module_id, *, catenate=True, aspect=None):
     return convert_name(name, suffix=suffix, catenate=catenate)
 
 
-def get_pointer_backend_name(id, module_id, *, catenate=False):
+def get_pointer_backend_name(id, module_id, *, catenate=False, aspect=None):
+    if aspect is None:
+        aspect = 'table'
+
+    if aspect not in ('table', 'index'):
+        raise ValueError(
+            f'unexpected aspect for pointer backend name: {aspect!r}')
+
     name = s_name.Name(module=str(module_id), name=str(id))
-    return convert_name(name, suffix='', catenate=catenate)
+
+    if aspect != 'table':
```

Comment:
```suggestion
    if aspect == 'index':
```

Reply:
No, I like the current formulation better.  Only the `table` aspect does not require a suffix, but reversing the condition would basically require amending it every time a new aspect is added.